stellar_sdk/operation/account_merge.py

```diff
@@ -29,7 +29,7 @@ def type_code(cls) -> int:
         return Xdr.const.ACCOUNT_MERGE
 
     def _to_operation_body(self) -> Xdr.nullclass:
-        destination = Keypair.from_public_key(self.destination).xdr_account_id()
+        destination = Keypair.from_public_key(self.destination).xdr_muxed_account()
```

Comment:
@overcat could you extend this to support `M` accounts as destination too? see https://github.com/stellar/js-stellar-base/commit/4009c3ad3309707d601b1bb39ace17adc87c2c64

Reply:
This has been fixed in #306.

https://github.com/StellarCN/py-stellar-base/blob/dev/stellar_sdk/operation/account_merge.py#L31